src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -162,12 +162,14 @@ class Microsoft::Terminal::Core::Terminal final :
     std::unique_ptr<::Microsoft::Console::VirtualTerminal::TerminalInput> _terminalInput;
 
     std::wstring _title;
+    std::wstring_view _startingTitle;

```

Comment:
I'm _mildly concerned_ about storing this as a `wstring_view`. Won't this explode when the method that sets this goes out of scope, as the backing `wstring` gets freed?